registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java

```diff
@@ -0,0 +1,8 @@
+package de.adorsys.registry.manager.service.exception;
+
+public class IbanException extends RuntimeException {
```

Comment:
have you added this exception to the advisor?

Reply:
I know. I want to create a separate task for adjusting the exception handling.

Reply:
ok, just do not forget about that